src/unix/linux-core.c

```diff
@@ -193,9 +193,11 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
   int count;
   int nfds;
   int fd;
-  int op;
+  int op = 0;
   int i;
```

Comment:
Why are you initializing these variables all of a sudden? Seems unrelated to your main change (the variables are not used anywhere in your changes). Are you getting compiler warnings?